docker-compose.yml

```diff
@@ -2,10 +2,10 @@ services:
   database:
     hostname: database
     image: postgres
-    env_file: .env
+    env_file: ${ENV_FILE:-.env.example}
```

Comment:
Shouldn't we use `.env`? Since `.env.example` is a file example. Also, didn't know about default values, really nice!